data/src/main/java/com/antosik/benchproject/data/movie/details/entity/GenresEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.antosik.benchproject.data.movie.details.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.antosik.benchproject.domain.movie.details.entity.GenresModel
+
+@Entity
+internal data class GenresEntity(
+    @PrimaryKey(autoGenerate = false)
```

Comment:
Isn't `autogenerate = false` a default value?

Reply:
From documentation 
`Returns: Whether the primary key should be auto-generated by SQLite or not. Defaults to false.`
So I deleted it